code/game/jobs/wages.dm

```diff
@@ -1,8 +1,8 @@
 //Determines starting account balance
 /datum/job/proc/one_time_payment(var/custom_factor = 1)
 	if (initial_balance != -1)
-		return round (initial_balance * RAND_DECIMAL(0.7, 1.3))
-	return round(wage * RAND_DECIMAL(1.5, 3.5))
+		return round (initial_balance * RAND_DECIMAL(1.4, 1.7))
+	return round(wage * rand(1.5 * 10, 1.5 * 10) / 10)
```

Comment:
```suggestion
		return round (initial_balance * rand(14, 17) / 10)
	return round(wage * rand(15, 35) / 10)
```
The proc rand() only produces integers and RAND_DECIMAL() doesn't work like we'd expect (or at all). For line 5, this is a better way to achieve what I'm assuming you intended.

Reply:
round(wage * rand(1.5 * 10, 1.5 * 10) / 10) should work the same as the adjusted RAND_DECIMAL

but in both cases a better idea would be actually fixing RAND_DECIMAL, and using that for both

Reply:
My point was that rand(1.5 * 10, 3.5 * 10) is identical in output to rand(15, 35), but with more operations. The latter is better practice. Also, I'd rather a RAND_DECIMAL() fix be its own PR.

Reply:
> My point was that rand(1.5 * 10, 3.5 * 10) is identical in output to rand(15, 35), but with more operations. The latter is better practice. Also, I'd rather a RAND_DECIMAL() fix be its own PR.

👍